resources/assets/js/annotations/components/annotationCanvas/magicWandInteraction.vue

```diff
@@ -60,18 +60,30 @@ export default {
             }
         }
     },
+    watch:{
+        isMagicWanding(isMagicWanding) {
+            if (this.canAdd) {
+                this.toggleMagicWandInteraction(isMagicWanding);
+            }
```

Comment:
No need for the `if` here because it's handled in `toggleMagicWand()`.

```suggestion
            this.toggleMagicWandInteraction(isMagicWanding);
```